lib/tests/test_shape_utils.py

```diff
@@ -140,3 +146,47 @@ def test_drop_parts_of_geoms(self, capsys, europe_gdf):
             europe_gdf[europe_gdf.name == 'Highlands and Islands'].area
             > gdf[gdf.name == 'Highlands and Islands'].area
         ).all()
+
+class TestPolygonsFromPoints:
+
+    @pytest.fixture
+    def points(self):
+        points = {  # Latitiude, Longitude
+            'athens': [37.9838, 23.7275],
+            'davos': [46.8027, 9.8360],
+            'stockholm': [59.3293, 18.0686],
+            'wurzburg': [49.7913, 9.9534]
+        }
+        areas = {
+            'athens': 1,
+            'davos': 5,
+            'stockholm': 100,
+            'wurzburg': 500
+        }
+        point_gdf = gpd.GeoDataFrame(
+            # x = longitude, y = latitude
+            geometry=gpd.points_from_xy([i[1] for i in points.values()], [i[0] for i in points.values()]),
```

Comment:
It isn't quite, because coordinates are usually communicated as `[lat, lon]`, but `[x, y]` is equivalent to `[lon, lat]`. Anyway, I'll update to be less verbose